libmeepgeom/cyl-ellipsoid-ll.cpp

```diff
@@ -97,25 +144,50 @@ int main(int argc, char *argv[])
   gaussian_src_time src(fcen, df);
   vec src_point = vec(0.0, 0.0);
   vec src_size  = vec(10.0, 10.0);
-  f.add_volume_source(src_cmpt, src, volume(src_point, src_size));
+  f.add_point_source(src_cmpt, src, src_point);
+
+  // first test: write permittivity to HDF5 file and
+  // compare with contents of reference file
+  if ( am_really_master() )
+   { f.output_hdf5(Dielectric, f.total_volume());
+     #define DATAFILE "eps-000000000.h5"
+     #define DATASET  "eps"
+     if (compare_hdf5_datasets(DATAFILE, DATASET, eps_ref_file, DATASET))
```

Comment:
don't use preprocessor symbols for "local variables" ... use `const char x[] = "..."`, or in this case I would just paste in the strings directly.